pkg/kf/commands/install/gke.go

```diff
@@ -0,0 +1,908 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+const (
+	KnativeBuildYAML      = "https://github.com/knative/build/releases/download/v0.7.0/build.yaml"
+	KfNightlyBuildYAML    = "https://storage.googleapis.com/artifacts.kf-releases.appspot.com/nightly-builds/releases/release-latest.yaml"
+	GKEClusterVersion     = "1.13.7-gke.8"
+	MachineType           = "n1-standard-4"
+	ImageType             = "COS"
+	DiskType              = "pd-standard"
+	DiskSize              = "100"
+	NumNodes              = "3"
+	DefaultMaxPodsPerNode = "110"
+	Addons                = "HorizontalPodAutoscaling,HttpLoadBalancing,Istio,CloudRun"
+)
+
+var (
+	prefixColor = color.New(color.FgHiBlue, color.Bold)
+	labelColor  = color.New(color.FgHiYellow, color.Bold)
+)
+
+// NewGKECommand creates a command that can install kf to GKE+Cloud Run.
+// TODO: This installer is using gcloud and kubectl under the hood. We should
+// really replace gcloud with google.golang.org/api and kubectl with
+// k8s.io/client-go
+func NewGKECommand() *cobra.Command {
+	var verbose bool
+
+	cmd := &cobra.Command{
+		Use:   "gke [subcommand]",
+		Short: "Install kf to GKE with Cloud Run",
+		Long: `
+This interactive installer will walk you through a process to install GKE with
+Cloud Run and then kf. Its really just running scripts under the hood,
+therefore you have to have gcloud and kubectl installed and available in the
+path.`,
```

Comment:
Change `Its really just running scripts under the hood,
therefore you have to have gcloud and kubectl installed and available in the
path.` to `You MUST have gcloud and kubectl installed and available on the
path.`